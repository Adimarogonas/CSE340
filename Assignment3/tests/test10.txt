{
    if (> r s) {
        r = * s t;
    }
    if (= t u) {
        t = + u v;
    }
    v = 0.0;
}
