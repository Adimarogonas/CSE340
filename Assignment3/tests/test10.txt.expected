r: real #
s: real #
t: real #
u: real #
v: real #
